WGLMakie/src/wglmakie.js

```diff
@@ -129,6 +128,9 @@ function threejs_module(canvas, comm, width, height, resize_to_body) {
         // we return nothing which will be handled by caller
         return;
     }
+    const pixel_ratio = window.devicePixelRatio || 1.0;
```

Comment:
Because you renamed this to `pixel_ratio` from `pixelRatio` there are a few references to `pixelRatio` that need to be updated in this file, see for instance the definition of `resize_callback`